pkg/controllers/job/job_controller.go

```diff
@@ -197,7 +198,10 @@ func (cc *Controller) Run(stopCh <-chan struct{}) {
 		cc.svcSynced, cc.cmdSynced, cc.pvcSynced, cc.pcSynced)
 
 	go wait.Until(cc.handleCommands, 0, stopCh)
-	go wait.Until(cc.worker, 0, stopCh)
+
+	for i := 0; i < workers; i++ {
+		go wait.Until(cc.worker, time.Second, stopCh)
```

Comment:
Why we use `time.Second` here but not `0` like the old logic? ...

Reply:
@lminzhw  followed k8s controller mgr controllers way.I hope if we have many controllers then its good to have some time instead of zero. may be we can have this parameter also as config parameter. what do you say?

Reply:
I browse some code of controller in k8s, and realize that most of k8s controller use `time.Second` as interval and multi-worker to solve the large scale problem.

Seems good :)